www/chrome.cast.js

```diff
@@ -544,7 +544,7 @@ var _session;
  * @param  {function} errorCallback
  */
 chrome.cast.initialize = function (apiConfig, successCallback, errorCallback) {
-    execute('initialize', apiConfig.sessionRequest.appId, apiConfig.autoJoinPolicy, apiConfig.defaultActionPolicy, function (err) {
+    execute('initialize', apiConfig.autoJoinPolicy, apiConfig.defaultActionPolicy, function (err) {
```

Comment:
What is the reason the `apiConfig.sessionRequest.appId` can be removed here? If I read the documentation, it states that it's value cannot be null, but is it safe to say we do not need te define it anymore in this 'initialize' method? Is it derived from somewhere else in the project? See: https://developers.google.com/cast/docs/reference/web_sender/chrome.cast.ApiConfig#ApiConfig

Reply:
Good question! The Cordova Cast Plugin attempts to mimic the cast sender web SDK API. The `chrome.cast.SessionRequest` instance expects an `appId`, but we no longer use it in the native code.

This way there is no breaking change in the plugin API and it still aligns with the web sender SDK API 😄